### DISCUSSION TITLE
I can't publish my website

### DISCUSSION THREAD

**Discussion by AJBernados** [community/community#78481] [New to GitHub]
How do I publish my website? Tutorials can't help me since there is supposed to be something here. Also, I don't want a custom domain.
![Sceenshot](https://github.com/community/community/assets/143686577/96c44128-aa41-41c4-b983-4c70619d65aa)

 :(

**Comment by ft10101** [comment]
when in the pages section, try changing source from "GitHub Actions" to "Deploy from a branch", then try again
